pynumaflow/function/server.py

```diff
@@ -284,7 +284,12 @@ def IsReady(
 
     async def __serve_async(self, server) -> None:
         udfunction_pb2_grpc.add_UserDefinedFunctionServicer_to_server(
-            UserDefinedFunctionServicer(self.__map_handler, self.__reduce_handler), server
+            UserDefinedFunctionServicer(
+                map_handler=self.__map_handler,
+                mapt_handler=self.__mapt_handler,
```

Comment:
+1. I totally missed updating this part when adding mapt handler. How did you find this mismatch? I wonder if we can add some unit tests to automatically detect such miss in the future.

Reply:
I was testing a reduce UDF, at that point it was erroring out. 
Yes definitely, should add some Unit tests/regression tests to increase coverage for such issues.